src/components/organisms/ConversationWindow.vue

```diff
@@ -1,71 +1,25 @@
 <template>
-  <div class="w-full h-full flex flex-col">
-    <div class="flex border-b border-gray-infomd p-2">
-      <img
-        src="../../assets/UserAccount/VirtualConcierge.svg"
-        alt="Bot Image."
-        class="h-9 mt-auto w-10"
-      />
-      <span class="font-bold pt-2 pl-2">Virtual Concierge</span>
+    <div class="flex">
+        <img src="../../assets/logo.png" alt="Bot Image." class="h-10 mt-auto w-10">
+        <div class="w-full pr-6 overflow-auto max-h-96 flex flex-col-reverse">
+            <!-- New text dialogs need to be added at the top since the div is flex reverse -->
+            <ConversationMessage :isUser=false :text="'Oh... Well, this is akward.'" />
+            <ConversationMessage :isUser=true :text="'I meant I am working too hard; I am all fired up right now.'"/>
+            <ConversationMessage :isUser=false :text="'I appreciate your concern. Apply water vigorously and call 911.'" />                
+            <ConversationMessage :isUser=true :text="'I need help! My desk is on fire!'" />
+            <ConversationMessage :isUser=false :text="'Hi, how can I help you?'" />
+            <!-- duplicate for content placeholder -->
+            <ConversationMessage :isUser=false :text="'Oh... Well, this is akward.'" />
+            <ConversationMessage :isUser=true :text="'I meant I am working too hard; I am all fired up right now.'"/>
+            <ConversationMessage :isUser=false :text="'I appreciate your concern. Apply water vigorously and call 911.'" />                
+            <ConversationMessage :isUser=true :text="'I need help! My desk is on fire!'" />
+            <ConversationMessage :isUser=false :text="'Hi, how can I help you?'" />
+            <!-- here -->
```

Comment:
I think I added this, but could you remove it in your PR please? :) The "here" comment